ert_shared/ensemble_evaluator/entity/unix_step.py

```diff
@@ -48,27 +48,36 @@ def run_job(self, client: Client, job: Any, run_path: Path):
 
         if cmd_exec.returncode != 0:
             self.logger.error(cmd_exec.stderr)
-            client.send_event(
-                ev_type=ids.EVTYPE_FM_JOB_FAILURE,
-                ev_source=job.get_source(self._ee_id),
-                ev_data={ids.ERROR_MSG: cmd_exec.stderr},
-            )
+            with Client(
+                prefect.context.url, prefect.context.token, prefect.context.cert
+            ) as client:
+                client.send_event(
+                    ev_type=ids.EVTYPE_FM_JOB_FAILURE,
+                    ev_source=job.get_source(self._ee_id),
+                    ev_data={ids.ERROR_MSG: cmd_exec.stderr},
+                )
```

Comment:
We could wrap this in a send_event function that takes the type, source and data, since we reuse this piece of code a bit now. But I am also fine with leaving it as is